retro/sadtalker.py

```diff
@@ -136,6 +176,26 @@ def sadtalker(
             inputs.source_image,
             os.path.join(save_dir, "face" + os.path.splitext(inputs.source_image)[1]),
         )
+        # convert image to jpg (to remove transparency) and make smaller than MAX_RES
+        if face_mime_type.startswith("image/"):
```

Comment:
Wondering if you can also do this on video and grab one frame (to fix this: https://app.asana.com/0/1203067047205953/1207725527090500)

Reply:
Good idea. I’ll do that

Reply:
[79228a9](https://github.com/GooeyAI/gooey-gpu/pull/26/commits/79228a9bd4ff4886849a8ea4e0f2ea4e78bd6286)